torch/optim/nadam.py

```diff
@@ -25,6 +26,8 @@
 
 
 class NAdam(Optimizer):
+    r"""Implementation for the NAdam algorithm."""
```

Comment:
we intentionally set the doc for this class later to give ourselves more freedom in using fstrings.

Reply:
This is true for all other optimizer files as well (other than the base)

Reply:
I will wait for a response on this before reviewing the rest of the optimizers.

Reply:
Thanks @janeyx99 I have ignored it in this Optim specific changes, https://github.com/pytorch/pytorch/pull/129086